var/spack/packages/flux/package.py

```diff
@@ -0,0 +1,36 @@
+from spack import *
+import os
+
+class Flux(Package):
+    """ A next-generation resource manager (pre-alpha) """
+
+    homepage = "https://github.com/flux-framework/flux-core"
+    url      = "https://github.com/flux-framework/flux-core"
+
+    version('master', branch='master', git='https://github.com/flux-framework/flux-core')
+
+    # Also needs autotools, but should use the system version if available
+    depends_on("zeromq@4.0.4:")
+    depends_on("czmq@2.2:")
+    depends_on("lua@5.1:5.1.99")
```

Comment:
It should work, but flux can't tolerate lua 5.2+, so I wanted to make sure it was only within the 5.1 range.  Is there a better way to mask off the high end?